### PR TITLE
Hlr pdf test md5

### PR DESCRIPTION
**Pull request by charleystran**
## Description of change

This PR adds a test for the HLR PDF generation to match the content with MD5 of an expected generated pdf. Also, the time in UTC has been added to the bottom PDF stamp.

### CODE REVIEW

File: modules/appeals_api/spec/fixtures/valid_200996_headers.json [hunk 1]
```diff
@@ -6,5 +6,7 @@
   "X-VA-Birth-Date": "1969-12-31",
   "X-VA-File-Number": "987654321",
   "X-VA-Service-Number": "876543210",
-  "X-VA-Insurance-Policy-Number": "987654321123456789"
+  "X-VA-Insurance-Policy-Number": "987654321123456789",
+  "X-Consumer-Username": "va.gov",
+  "X-Consumer-ID": "some-guid"
```
Comment (edmangimelli) [inline modules/appeals_api/spec/fixtures/valid_200996_headers.json:11]: consumer-id is automatically added by Kong, but is consumer-username? (i.e. does this need to be in the documentation)

File: modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb [hunk 2]
```diff
@@ -78,6 +78,18 @@
     end
   end
 
+  context 'pdf content verification' do
+    it 'generates the expected pdf' do
+      Timecop.freeze(Time.zone.parse('2020-01-01T08:00:00Z'))
+      path = described_class.new.generate_pdf(higher_level_review.id)
```
Comment (edmangimelli) [inline modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb:84]: does this pdf need to be cleaned up post-test?

File: modules/appeals_api/spec/fixtures/valid_200996_headers.json (hunk 1 above)
Comment (charleystran) [reply modules/appeals_api/spec/fixtures/valid_200996_headers.json:11]: they are both auto generated by kong.

File: modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb (hunk 2 above)
Comment (charleystran) [reply modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb:84]: its writing to tmp, but we could destroy it to be safe

Comment (edmangimelli) [reply modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb:84]: you're call

File: modules/appeals_api/lib/appeals_api/higher_level_review_pdf_constructor.rb [hunk 3]
```diff
@@ -21,7 +21,13 @@ def fill_pdf
     end
 
     def stamp_pdf(pdf_path, consumer_name)
-      bottom_stamped_path = CentralMail::DatestampPdf.new(pdf_path).run(text: 'API.VA.GOV', x: 5, y: 5)
+      stamper = CentralMail::DatestampPdf.new(pdf_path)
+      bottom_stamped_path = stamper.run(
+        text: "API.VA.GOV #{Time.zone.now.utc.strftime('%Y-%m-%d %H:%M%Z')}",
```
Comment (edmangimelli) [inline modules/appeals_api/lib/appeals_api/higher_level_review_pdf_constructor.rb:26]: this timestamp is being discussed
https://lighthouseva.slack.com/archives/CT0G8GMS7/p1588287510338500?thread_ts=1588271316.328000&cid=CT0G8GMS7
but I'm thinking my concerns relate more to the date that accompanies the signature

**Pull request merged**